GitUI/UserControls/BlameControl.cs

```diff
@@ -332,14 +345,46 @@ private void ProcessBlame(string filename, GitRevision revision, IReadOnlyList<O
             var lineBuilder = new StringBuilder(lineLength + 2);
             var gutter = new StringBuilder(capacity: lineBuilder.Capacity * _blame.Lines.Count);
             var emptyLine = new string(' ', lineLength);
-            foreach (var line in _blame.Lines)
+            var cacheAvatars = new Dictionary<string, Image>();
+            var noAuthorTask = (Image)new Bitmap(Images.User80, avatarSize, avatarSize);
+            for (var index = 0; index < _blame.Lines.Count; index++)
             {
+                var line = _blame.Lines[index];
                 if (line.Commit == lastCommit)
                 {
+                    if (gitBlameDisplays != null)
+                    {
+                        gitBlameDisplays[index].Avatar = null;
```

Comment:
Are we certain there will be no entries in the collection with `.Avatar == null`?
I feel we must re-iterate the collection to ensure this before it gets return.

Reply:
That's the goal to have the avatar property to be null to not display an avatar on the corresponding line (when we are on the same commit):

![image](https://user-images.githubusercontent.com/460196/78995472-b3092380-7b42-11ea-9619-f8bd93f473bd.png)

But the initial value is already `null`, so this code is not needed. I removed it.